metaflow/plugins/pypi/micromamba.py

```diff
@@ -338,23 +339,37 @@ def _install_micromamba(installation_location):
     # Unfortunately no 32bit binaries are available for micromamba, which ideally
     # shouldn't be much of a problem in today's world.
     platform = conda_platform()
-    try:
-        subprocess.Popen(f"mkdir -p {installation_location}", shell=True).wait()
```

Comment:
do we need to create the install location?

Reply:
I swapped this to `os.makedirs`, can also revert and instead keep the mkdir, or alternatively supply `--create-dirs` for curl.

Reply:
but yes, piping the curl to tar can fail if the install location does not exist, as tar doesn't create missing target directories by itself

Reply:
ah i missed the os.makedirs